Modules/Projects/Core/Script_MiscUtilities.lua

```diff
@@ -44,6 +44,27 @@ function LogGsub(key, vars, color)
     LogRaw(strval, color)
 end
 
+function PreySense(pred, style, does_digest)
+    if(Player.PreySense)then
+    local color
+    local vtype 
+        if style == 0 then vtype = "OV" 
+        elseif style == 1 then vtype = "AV" 
+        elseif style == 2 then vtype = "CV" 
+        elseif style == 3 then vtype = "UB" 
+        end
+    local vfinal
+        if fatal then 
```

Comment:
There is no local variable named `fatal` anymore; you probably meant to write `does_digest` here.